internal/testutils/db.go

```diff
@@ -130,7 +132,8 @@ func dbToCsv(t *testing.T, db *sql.DB, w io.Writer, usePredicatableFieldValues b
 		err = dumpTable(t, db, tableName, w, usePredicatableFieldValues)
 		require.NoError(t, err, "Failed to dump table %s", tableName)
 
-		separateTables = true
+		_, err = w.Write([]byte("\n"))
```

Comment:
Ok. I thought it would be better to put it in a different PR, but I needed those changes in order to make the other functions work, but I should have opened another PR and requested it to be merged before this one.